localtypings/pxtarget.d.ts

```diff
@@ -327,6 +327,7 @@ declare namespace pxt {
         qrCode?: boolean; // generate QR code for shared urls
         importExtensionFiles?: boolean; // import extensions from files
         debugExtensionCode?: boolean; // debug extension and libs code in the Monaco debugger
+        unsupportedBrowsers?: string[] // list of unsupported browsers for a specific target (eg IE11 in arcade). check browserutils.js browser() function for strings
```

Comment:
Move this into the apptarget - it needs to be available in all pages and we delete theme in some cases (see targetlight.js)